app/main.py

```diff
@@ -1,5 +1,82 @@
+from __future__ import annotations
+
+
 class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        if coords is None:
+            coords = [0, 0]
+        self.name = name
+        self.weight = weight
+        self.coords = coords
```

Comment:
Do not use multi-line if condition to define `coords` make it in one line as this
```suggestion
        self.name = name
        self.weight = weight
        self.coords = coords or [0, 0]
```
or as this 
```suggestion
        self.name = name
        self.weight = weight
        self.coords = coords if coords else [0, 0]
```
Make this change to all places where applicable.